Sources/HTTPManager.swift

```diff
@@ -1620,6 +1620,10 @@ extension HTTPManager {
             switch uploadBody {
             case .data(let data)?:
                 networkTask = inner.session.uploadTask(with: urlRequest, from: data)
+            case .formUrlEncoded(let queryItems)?:
+                networkTask = inner.session.uploadTask(with: urlRequest, from: FormURLEncoded.data(for: queryItems))
+            case .json(let json)?:
+                networkTask = inner.session.uploadTask(with: urlRequest, from: JSON.encodeAsData(json))
```

Comment:
Got it; I found this when testing against a Docker instance of our environment; doesn't happen on the real server.